### PR TITLE
Enables DirectionalSpriteV2 for mobs

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Modifies mob scripts to allow the use of `DirectionalSpriteV2`. Partially implements #5250.
    - This means mobs can use `SpriteHandler`.
    - Makes two mob animation components (`MobAnimator` and `AnimationLooper`) deprecated and so they have been removed. 
- Adds mob burning and electrocuted overlays.
    - The electrocuted overlay currently uses the humanoid prefab.
- Adds or tweaks mob namespaces.

> Note: there will be a few harmless errors until issue #5325 is done.

### CODE REVIEW

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+				electrocutedOverlay = Instantiate(electrocutedPrefab, transform).GetComponent<PlayerDirectionalOverlay>();
+				electrocutedOverlay.enabled = true;
+				electrocutedOverlay.StopOverlay();
+			}
+		}
+
+		#endregion Lifecycle
+
+		//The local rotation of the sprite obj
+		private void SyncRotChange(float oldRot, float newRot)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/NPC/MobSprite.cs:95]: Codacy found an issue: [Remove this unused method parameter 'oldRot'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172)

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/NPC/MobSprite.cs  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172&bid=20635757)

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+		{
+			if (burningOverlay == null)
+			{
+				AddOverlayGameObjects();
+			}
+
+			burningOverlay.StartOverlay(directional.CurrentDirection);
+		}
+
+		private void OnDirectionChange(Orientation newDirection)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/NPC/MobSprite.cs:169]: Codacy found an issue: [Remove this unused method parameter 'newDirection'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172)

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/NPC/MobSprite.cs:169]: `SyncVar`.

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs (hunk 1 above)
Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/NPC/MobSprite.cs:95]: `SyncVar`.

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs (hunk 2 above)
Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/NPC/MobSprite.cs:169]: This is an event. Cannot remove the parameter.

**Comment by Bod9001** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- UnityProject/Assets/Scripts/NPC/MobSprite.cs  3
- UnityProject/Assets/Scripts/Health/Living/SimpleAnimal/SimpleAnimal.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172&bid=20635757)

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+				electrocutedOverlay = Instantiate(electrocutedPrefab, transform).GetComponent<PlayerDirectionalOverlay>();
+				electrocutedOverlay.enabled = true;
+				electrocutedOverlay.StopOverlay();
+			}
+		}
+
+		#endregion Lifecycle
+
+		//The local rotation of the sprite obj
+		private void SyncRotChange(float oldRot, float newRot)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/NPC/MobSprite.cs:95]: Codacy found an issue: [Remove this unused method parameter 'oldRot'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172)

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+		{
+			if (burningOverlay == null)
+			{
+				AddOverlayGameObjects();
+			}
+
+			burningOverlay.StartOverlay(directional.CurrentDirection);
+		}
+
+		private void OnDirectionChange(Orientation newDirection)
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/NPC/MobSprite.cs:175]: Codacy found an issue: [Remove this unused method parameter 'newDirection'.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6356172)

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs (hunk 3 above)
Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/NPC/MobSprite.cs:95]: `SyncVar`.

File: UnityProject/Assets/Scripts/NPC/MobSprite.cs (hunk 4 above)
Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/NPC/MobSprite.cs:175]: Event listener.

**Pull request merged**